examples/kops-with-vpc/README.md

```diff
@@ -0,0 +1,184 @@
+# Example to test the Kops workflow with an existing VPC on AWS
+
+## Summary
+
+We first use `terraform` to create:
+1. a VPC,
+2. it's subnets and
+3. an internet gateway, along with route tables for local and internet traffic.
+
+We then use `kops` to create the kubernetes cluster in this existing VPC, where `kops`, by itself, manages the creation of:
+1. DNS records under the base FQDN,
+2. The auto-scaling group, and its launch configurations
+3. EBS volumes
+4. IAM instance profiles, role policies
+5. Keypairs for every kubernetes component
+6. Files and secrets required to configu kubernetes clusters
+7. Security groups
+...and modifies the tags on existing subnets to make things easier for discovery within the kubernetes cluster
+
+## Environment creation and deployment
+`NOTE`: For usage with `aws-env` have each command below execute in the form `aws-env -p <your-profile> -- <the-command>`.
+
+### Part 1: Terraform
+
+```
+pushd terraform
+source prod/env
```

Comment:
I'd suggest providing a `template/env` instead of `prod/env`, and then have instructions to copy and modify the template (same applies in the kubernetes part).